src/main/java/pvs/app/service/thread/GitlabCommitLoaderThread.java

```diff
@@ -0,0 +1,65 @@
+package pvs.app.service.thread;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.gitlab4j.api.models.CommitStats;
+import org.gitlab4j.api.utils.JacksonJson;
+import org.springframework.web.reactive.function.client.WebClient;
+import pvs.app.dto.GitlabCommitDTO;
+import pvs.app.service.GitlabApiService;
+import pvs.app.service.GitlabCommitService;
+
+import java.io.IOException;
+import java.util.*;
```

Comment:
Since there only use `Optional`, we don't need to import all things from package.
```suggestion
import java.util.Optional;
```